tests/automation/group_disappearing_messages.spec.ts

```diff
@@ -49,12 +49,10 @@ mediaArray.forEach(({ mediaType, path }) => {
       } else {
         await sendMedia(aliceWindow1, path, testMessage);
       }
-      await Promise.all([
-        waitForLoadingAnimationToFinish(bobWindow1, 'loading-animation'),
-        waitForLoadingAnimationToFinish(charlieWindow1, 'loading-animation'),
-      ]);
```

Comment:
move those down for each type because voice message attachment spinner is shown after the message body is found.